File: Lab01/C_Lang/src/Demo11.cpp
//
//  Created by NDHuy at 12/04/2020
//  Copyright 2020 by SEstudio
//  Check symmetry number
//

#include <stdio.h>

int main()
{
    int n; 
    printf("Enter n > 0: ");
    scanf("%d", &n);

    if (n < 0)
        printf("%d is not greater than 0", n);
    else
    {
        int t; 
        int m = 0; 
        int k = n;

        // Method-01
        // do
        // {
        //     t = k % 10;
        //     m = (m * 10) + t;
        //     k = k / 10;
        // } while (k != 0);

        // Method-02
        for(int i = n; i != 0; i /= 10)
        {
            m = (m * 10) + (i % 10);
        }
        
        printf("Symmetry number: %d \n", m);
        
        if(n == m)
            printf("%d is symmetry number\n", n);
        else
            printf("%d is not symmetry number\n", n);
    }
    
    return 0;
}

File: 01_BasicPagramming/Lab01/Python_Lang/Demo11.py
#
#  Created by NDHuy at 12/04/2020
#  Copyright 2020 by SEstudio
#  Check symmetry number
#

def main():
    n = int(input("Enter n > 0: "))

    if n >= 0:
        i = n
        m = 0
        while i != 0:
            m = (m * 10) + (i % 10)
            i = int(i / 10)

        print("Symmetry number: %d" %(m))

        if n == m:
            print("%d is symmetry number" %(n))
        else:
            print("%d is not symmetry number" %(n))
    else:
        print("%d is not greate than Zero" %(n))


if __name__ == "__main__":
    main()